projects/VRDR/IJEMortality.cs

```diff
@@ -832,7 +832,7 @@ public string AGETYPE
             {
                 // Pull code from coded unit.   "code" field is not required by VRDR IG
                 string code = Dictionary_Get_Full("AGETYPE", "AgeAtDeath", "code") ?? "";
-                Mappings.UnitsOfAge.FHIRToIJE.TryGetValue(code, out string ijeValue);
+                VR.Mappings.UnitsOfAge.FHIRToIJE.TryGetValue(code, out string ijeValue);
```

Comment:
Should mappings still use VRDR.Mappings instead of VR to stay aligned with STU 2.2?

Reply:
Yes, fixed